Network/Socket/ByteString/IO.hsc

```diff
@@ -231,7 +231,7 @@ sendManyWithFds s bss fds =
             sendBufMsg s addr bufsizs cmsgs flags
   where
     addr = NullSockAddr
-    cmsgs = encodeCmsg <$> fds
+    cmsgs = encodeCmsg . (:[]) <$> fds
```

Comment:
This could now send all the file descriptors in a single `Cmsg`, but I left it this way to avoid changing the current behaviour.